webrender/src/prim_store.rs

```diff
@@ -970,6 +973,7 @@ impl PrimitiveStore {
                     local_rect: *local_rect,
```

Comment:
given how often this struct gets repeated, we should make one before the `match` and then do
```rust
let metadata = PrimitiveMetadata {
    cpu_prim_index: SpecificPrimitiveIndex(self.cpu_yuv_images.len()),
    .. base_metadata
};
```